agents-manage-ui/src/components/graph/sidepane/nodes/model-selector.tsx

```diff
@@ -45,7 +45,8 @@ export function ModelSelector({
       const model = models.find((m) => m.value === value);
       if (model) return model;
     }
-    return value ? { value, label: value } : null;
+    // Handle custom models
+    return value ? { value, label: `${value} (custom)` } : null;
```

Comment:
@tim-inkeep does this label still make sense given the scope change from custom models to just gateway and open router?